easybuild/easyconfigs/r/R/R-3.6.0-foss-2019a.eb

```diff
@@ -1962,7 +1961,7 @@ exts_list = [
         'checksums': ['6b82edf9fc7d1f98d5632bc94476bd10b39134a7bb267120b5e979a4590e195f'],
     }),
     ('hdf5r', '1.2.0', {
-        'preinstallopts': "unset LIBS && ",
```

Comment:
@fizwit Please keep the `preinstallopts`, you should have both...

Reply:
preinstallopts has been re-installed